.gitignore

```diff
@@ -3,6 +3,9 @@ terraform-provider-anxcloud
 bin
 .idea
 
+# Ignore direnv file as it contains secrets
+.envrc
```

Comment:
is this specific to anxcloud? otherwise it should land in your .gitignore_global aka core.excludesfile of your .gitconfig

Reply:
we also have it in go-anxcloud's .gitignore and since it contains secrets, better safe than sorry I guesss